extension/js/quick-reply.js

```diff
@@ -371,12 +371,27 @@ QuickReplyBox.prototype.fetchFormCookie = function(threadid) {
 };
 
 QuickReplyBox.prototype.updatePreview = function() {
-    if(jQuery('#post-message').val().length > 0) {
-        var parser = new PreviewParser(jQuery('#post-message').val(), this.sortedEmotes);
-        jQuery('#preview-content').html(parser.fetchResult());
+    let preview = document.querySelector('#preview-content');
```

Comment:
Small issue, but if you already have an ID, `document.getElementById()` should be used instead of `document.querySelector()`